functions/nvm.fish

```diff
@@ -189,21 +189,16 @@ function _nvm_use
         command mv -f $nvm_config/$ver. $target
     end
 
-    if test -s "$nvm_config/version"
-        read -l last <"$nvm_config/version"
-        if set -l i (contains -i -- "$nvm_config/$last/bin" $fish_user_paths)
-            set -e fish_user_paths[$i]
-        end
-    end
-
     if set -l root (_nvm_find_up (pwd) $nvm_file)
         echo $argv[1] >$root/$nvm_file
     end
 
     echo $ver >$nvm_config/version
 
-    if not contains -- "$nvm_config/$ver/bin" $fish_user_paths
-        set -U fish_user_paths "$nvm_config/$ver/bin" $fish_user_paths
+    ln -sfn "$nvm_config/$ver" "$nvm_config/current"
+
+    if not contains -- "$nvm_config/current/bin" $fish_user_paths
+        set -U fish_user_paths "$nvm_config/current/bin" $fish_user_paths
```

Comment:
@antitoxic What's the purpose of `/current/bin` and what is this `current` prefix that I haven't heard about?

Reply:
It's not a prefix that should be heard. Just a name. If you prefer it can be `inuse`, `used`, etc. It's a symlink to the currently in use nodejs .

Reply:
@antitoxic In other words, this is an arbitrary name?